traffic_ops/traffic_ops_golang/user/user.go

```diff
@@ -899,7 +899,8 @@ func Create(w http.ResponseWriter, r *http.Request) {
 		Response: userV4,
 		Alerts:   tc.CreateAlerts(tc.SuccessLevel, "user was created."),
 	}
-	w.Header().Set("Location", fmt.Sprintf("/api/4.0/users?id=%d", *userV4.ID))
+	version := fmt.Sprintf("%s.%s", strconv.FormatUint(inf.Version.Major, 10), strconv.FormatUint(inf.Version.Minor, 10))
```

Comment:
Since you're using `fmt.Sprintf` already, there's no need to bring `strconv` into it. Just use `%d` instead of `%s`.